src/utils/section-handler.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const all = require('mdast-util-to-hast/lib/all');
+const wrap = require('mdast-util-to-hast/lib/wrap');
+
+const DEFAULT_SECTION_TAG = 'div';
+const DEFAULT_SECTION_CLASS = 'hlx-section';
+const SYSTEM_META_PROPERTIES = ['class', 'meta', 'tagname', 'types'];
```

Comment:
YAML properties:
- `class {string}`: will allow overwriting the default section class (i.e. `hlx-section`)
- `meta {boolean}`: when set to `true` will output any additional YAML properties as `data-hlx-*` attributes
- `tagname {string}`: will allow overwriting the default section tag (i.e. `div`)
- `types {boolean}`: when set to `true` will output the section types as additional classes on the section (i.e. `has-image`, etc.)

Reply:
Since `meta` and `types` are actual feature flags, should we follow a specific naming convention?
`enable_meta`/`enable_types`, `output_meta`/`output_types`, etc.

Reply:
I would only do that once we expose it as a config parameter.